gradle-plugins/src/integTest/groovy/com/linkedin/pegasus/gradle/PegasusPluginIntegrationTest.groovy

```diff
@@ -22,7 +22,7 @@ class PegasusPluginIntegrationTest extends Specification {
         .withProjectDir(tempDir.root)
         .withPluginClasspath()
         .withArguments('mainDataTemplateJar')
-        .forwardOutput()
+        //.forwardOutput()
```

Comment:
Context?

Reply:
Sorry, it's unrelated to this PR. GradleRunner#forwardOutput includes the stdout of the Gradle project being tested in the CI logs. Useful for debugging, but is unnecessarily verbose once things are working as expected. I can omit this line to keep this PR "pure", but this change is also so insignificant I'm not sure it deserves its own PR!

Reply:
Yeah, it's no big deal. I personally like to leave comments about something like this, because it could be ambiguous to someone way down the road as to whether this was intentional or a mistake.

Reply:
Changing this line isn't worth the trouble; reverting.